Mathlib/Analysis/InnerProductSpace/LinearPMap.lean

```diff
@@ -268,3 +270,90 @@ theorem _root_.IsSelfAdjoint.dense_domain (hA : IsSelfAdjoint A) : Dense (A.doma
 end LinearPMap
 
 end Star
+
+/-! ### The graph of the adjoint -/
+
+namespace Submodule
+
+/-- The adjoint of a submodule -/
```

Comment:
This definition is a little bit misleading, as you are using the product space `E × F` which has the sup norm, but instead you're using the `L^2` distance. Options would be either to use `WithLp.prod 2 E F` (whatever the correct spelling) (but I am not sure it's a good idea, since you work with graphs all the time and these are defined using the regular product), or to emphasize this difficulty and the conscious design choice to still use `E × F` in the docstring.

Reply:
I would like to leave the code as it is for now, because everything for graphs is in terms of `E × F`. If it turns out that we use `WithLp` more, then I am happy to revisit this.